reactive/containerd.py

```diff
@@ -275,6 +276,17 @@ def config_changed():
     context['custom_registries'] = \
         merge_custom_registries(context['custom_registries'])
 
+    untrusted = DB.get('untrusted')
+    if untrusted:
+        context['untrusted'] = True
+        context['untrusted_name'] = untrusted['name']
+        context['untrusted_path'] = untrusted['binary_path']
```

Comment:
If binary_path has the full path including the executable, is it wrong to call this untrusted_path? This is obviously a nitpick type thing, but is it a path if it has an executable in it? I'm honestly not sure, but my assumption was this was a path without an executable name in it. like `/usr/local/bin` and not `/usr/local/bin/my_exe`. Maybe I just need to breathe more...

Reply:
I see where you're coming from, but I think a path can be a dir or file, no?